doc/devel/README.txt

```diff
@@ -0,0 +1,8 @@
+All documentation in the gitwash directory are automatically generated by running the below script in
+the project's root directory:
+
+python gitwash_dumper.py doc/devel Matplotlib --repo-name=matplotlib --github-user=matplotlib \
```

Comment:
I restructured the sentence a bit and added the link to gitwash_dumper.py script. Have a look.